plugins/Kaleidoscope-Devel-ArduinoTrace/src/kaleidoscope/plugin/ArduinoTrace.h

```diff
@@ -0,0 +1,45 @@
+/* -*- mode: c++ -*-
+ * Copyright (C) 2017, 2018  Keyboard.io, Inc
+ *
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation, version 3.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/*
+#pragma once
+
+#include "kaleidoscope/Runtime.h"
+
+#ifndef ARDUINOTRACE_SERIAL
+#define ARDUINOTRACE_SERIAL DebugStderr
+#endif
+
+#include "ArduinoTrace.h"
```

Comment:
Is it intended that the entire contents are commented out? If so, I'm not sure I understand how this plugin works...